functions/__sf_section_kubecontext.fish

```diff
@@ -0,0 +1,32 @@
+#
+#  Kubernetes (kubectl)
+#
+# Kubernetes is an open-source system for deployment, scaling,
+# and management of containerized applications.
+# Link: https://kubernetes.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+function __sf_section_kubecontext -d "Display the kubernetes context"
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_SHOW true
+	__sf_util_set_default SPACEFISH_KUBECONTEXT_PREFIX " "
```

Comment:
Looking at the spaceship implementation, the default prefix is `"at "`: https://github.com/denysdovhan/spaceship-prompt/blob/master/sections/kubecontext.zsh#L13